calcom/cal.com
# fix: save workflow button not visible on dev env

## What does this PR do?



This PR renders the "Save" button in edit/create-workflow flow on the development environment.

Fixes #14909 

## Type of change



- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

1. Click on "Workflows" in the sidebar
2. Click "+ Create a workflow" or edit a workflow.
3. On the top right corner, the "Save" button should be visible.

<img width="1440" alt="image" src="https://github.com/calcom/cal.com/assets/93566941/e44cd969-ddec-40c1-b0eb-b3eff7eea860">


## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
